keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -645,8 +797,8 @@ a different pod the next time (or none). This would leave a pod with a
 condition that it was preempted, when it actually wasn't. This in turn
 could lead to inproper handling of the pod by the job controller.
 
-As a solution we are going to implement a worker, added to the disruption
-controller which clears the pod condition added if `DeletionTimestamp` is
+As a solution we implement a worker, part of the disruption
+controller, which clears the pod condition added if `DeletionTimestamp` is
 not added to the pod for a long enough time (for example 2 minutes).
 
 #### Marking pods as Failed
```

Comment:
A pod might not be ready while still running. We want to count failed pods (stopped running). I think it's fair to set the phase from the podgc if the Node does not exist.

Reply:
Yes, I suppose that would make sense if the node does not exist. 

In general the pod should be deleted if the node does not exist, but if there is a finalizer as you mentioned, that won't work. In that case, I suppose it would make sense to set the phase as well (since kubelet would not be able to set it any longer).